Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM golang:1.10-alpine as rita-builder
+FROM golang:1.14-alpine as rita-builder
 
 RUN apk add --no-cache git make ca-certificates wget build-base
-RUN wget -q -O /go/bin/dep https://github.com/golang/dep/releases/download/v0.5.3/dep-linux-amd64 && chmod +x /go/bin/dep
 
 WORKDIR /go/src/github.com/activecm/rita
 
 # cache dependencies
-COPY Gopkg.lock Gopkg.toml Makefile ./
-RUN make vendor
```

Comment:
I trust that if it works with the current dependency set, that it would work with another one added. The docker build seems to work fine for me. If the dependencies weren't downloaded according the mod file, it would have broken. It helps that we always start from scratch in Docker, so there's less  to worry about.